site/pages/home/brie-slice/style.css

```diff
@@ -11,7 +11,7 @@
 .heading {
   composes: fontL from "../../../css/typography/_fonts.css";
   composes: serif from "../../../css/typography/_fonts.css";
-  max-width: contentMaxWidth;
+  max-width: 1120px;
```

Comment:
is this change required? I don't know what the `contentMaxWidth` was but might need a design review to check that is correct behaviour